attribute-utils/src/main/java/org/hypertrace/attribute/db/DbAttributeUtils.java

```diff
@@ -0,0 +1,156 @@
+import org.hypertrace.core.span.constants.v1.Redis;
+import org.hypertrace.core.span.constants.v1.Sql;
+
+public class DbAttributeUtils {
+
+  // db related OTEL attributes
+  private static final String OTEL_DB_SYSTEM = OTelDbAttributes.DB_SYSTEM.getValue();
+  private static final String OTEL_DB_CONNECTION_STRING = OTelDbAttributes.DB_CONNECTION_STRING.getValue();
+  private static final String OTEL_DB_OPERATION = OTelDbAttributes.DB_OPERATION.getValue();
+  private static final String OTEL_NET_PEER_IP = OTelDbAttributes.NET_PEER_IP.getValue();
+  private static final String OTEL_NET_PEER_PORT = OTelDbAttributes.NET_PEER_PORT.getValue();
+  private static final String OTEL_NET_PEER_NAME = OTelDbAttributes.NET_PEER_NAME.getValue();
+  private static final String OTEL_NET_TRANSPORT = OTelDbAttributes.NET_TRANSPORT.getValue();
+
+  // mongo specific attributes
+  private static final String OTEL_MONGO_DB_SYSTEM_VALUE = OTelDbAttributes.MONGODB_SYSTEM_VALUE.getValue();
+  private static final String OTHER_MONGO_ADDRESS = RawSpanConstants.getValue(Mongo.MONGO_ADDRESS);
+  private static final String OTHER_MONGO_URL = RawSpanConstants.getValue(Mongo.MONGO_URL);
+  private static final String OTHER_MONGO_OPERATION = RawSpanConstants.getValue(Mongo.MONGO_OPERATION);
+  private static final String OTHER_MONGO_NAMESPACE = RawSpanConstants.getValue(Mongo.MONGO_NAMESPACE);
+  private static final String OTEL_MONGO_COLLECTION = OTelDbAttributes.MONGODB_COLLECTION.getValue();
+  //private static final AttributeValue MONGO_DEFAULT_PORT = EnricherUtil.createAttributeValue(EnricherUtil.createAttributeValue("27017"));
+
+  // redis specific attributes
+  private static final String OTHER_REDIS_CONNECTION = RawSpanConstants.getValue(Redis.REDIS_CONNECTION);
+  private static final String OTEL_REDIS_DB_SYSTEM_VALUE = "redis";
+  //private static final AttributeValue REDIS_DEFAULT_PORT = EnricherUtil.createAttributeValue(EnricherUtil.createAttributeValue("6379"));
+
+  // sql specific attributes
+  private static final String[] OTEL_SQL_DB_SYSTEM_VALUES =
+      {
+          "mysql", "oracle", "mssql",
+          "other_sql", "db2", "postgresql",
+          "redshift", "hive", "cloudscape", "hsqldb"
+      };
+  private static final String JDBC_EVENT_PREFIX = "jdbc";
+  private static final String SQL_URL = RawSpanConstants.getValue(Sql.SQL_SQL_URL);
+
+  public static Optional<String> getMongoURI(Event event) {
+    if (SpanAttributeUtils.containsAttributeKey(event, OTHER_MONGO_ADDRESS)) {
+      return Optional.of(SpanAttributeUtils.getStringAttribute(event, OTHER_MONGO_ADDRESS));
+    } else if (SpanAttributeUtils.containsAttributeKey(event, OTHER_MONGO_URL)) {
+      return Optional.of(SpanAttributeUtils.getStringAttribute(event, OTHER_MONGO_URL));
+    } else if (SpanAttributeUtils.containsAttributeKey(event, OTEL_DB_SYSTEM)) {
+      if (!OTEL_MONGO_DB_SYSTEM_VALUE.equals(event.getAttributes().getAttributeMap().get(
```

Comment:
Can we use method `getStringAttribute` from SpanAttributeUtils for reading value?

Reply:
done